shuup/front/templates/shuup/front/macros/general.jinja

```diff
@@ -383,8 +383,12 @@
     {% if text %}
         <meta property="og:description" content="{{ text|safe|striptags|truncate(160)|replace("\n", " ")|replace("\"", "'") }}">
     {% endif %}
-    {% if primary_image %}
-        <meta property="og:image" content="{{ request.build_absolute_uri(primary_image.url) }}">
+    {% set product_image = object.primary_image or object.media.first() %}
```

Comment:
Are you sure this `object.media.first()` will return the first `IMAGE`?
I guess we have better solutions and some other utils like `ShopProduct.public_images`